eng/pipelines/common/templates/pipeline-with-resources.yml

```diff
@@ -15,9 +15,9 @@ resources:
         ROOTFS_DIR: /crossrootfs/armv6
 
     - container: linux_arm64
-      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-arm64
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-cross-arm64
       env:
-        ROOTFS_DIR: /crossrootfs/arm64
+        ROOTFS_DIR: /crossrootfs
```

Comment:
This convention is documented in number of places. Lets not introduce a break because everyone using these prereq images outside this CI (for years) remembers the default location of `ROOTFS_DIR`.

```
eng/pipelines/common/templates/pipeline-with-resources.yml:        ROOTFS_DIR: /crossrootfs/arm
eng/pipelines/common/templates/pipeline-with-resources.yml:        ROOTFS_DIR: /crossrootfs/armv6
eng/pipelines/common/templates/pipeline-with-resources.yml:        ROOTFS_DIR: /crossrootfs/arm64
eng/pipelines/common/templates/pipeline-with-resources.yml:        ROOTFS_DIR: /crossrootfs/arm
eng/pipelines/common/templates/pipeline-with-resources.yml:        ROOTFS_DIR: /crossrootfs/arm64
eng/pipelines/common/templates/pipeline-with-resources.yml:        ROOTFS_DIR: /crossrootfs/x86
eng/pipelines/common/templates/pipeline-with-resources.yml:        ROOTFS_DIR: /crossrootfs/s390x
eng/pipelines/common/templates/pipeline-with-resources.yml:        ROOTFS_DIR: /crossrootfs/ppc64le
eng/pipelines/common/templates/pipeline-with-resources.yml:        ROOTFS_DIR: /crossrootfs/x64
eng/pipelines/common/templates/pipeline-with-resources.yml:        ROOTFS_DIR: /crossrootfs/armel
```